spec/user_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it { should validate_presence_of(:name) }
+  it { should validate_numericality_of(:posts_counter).only_integer.is_greater_than_or_equal_to(0) }
+
```

Comment:
- As I pointed out earlier, you have the test written but you've not added the validation to the `user` model. please, kindly add them.

Reply:
### TEST FILES  STRUCTURE

- Hey, since you've written the test for your models, kindly create a model folder, to move all the spec files inside of it. your file structure should be `spec/models/user_spec.rb` to improve your test file structure.